cluster-autoscaler/cloudprovider/gke/gke_manager.go

```diff
@@ -351,8 +332,12 @@ func (m *gkeManagerImpl) createNodePool(mig gce.Mig) (gce.Mig, error) {
 	}
 	// TODO(aleksandra-malinowska): support multi-zonal node pools.
 	for _, existingMig := range m.cache.GetMigs() {
-		if existingMig.Config.NodePoolName() == mig.NodePoolName() {
-			return existingMig.Config, nil
+		gkeMig, ok := existingMig.Config.(*gkeMig)
+		if !ok {
+			glog.Fatalf("Really really bad, Mig is not gkeMig: got %v, want gkeMig", reflect.TypeOf(existingMig.Config))
```

Comment:
I think "Really really bad" is implied by Fatalf :)

Reply:
Fixed to use InternalError :)